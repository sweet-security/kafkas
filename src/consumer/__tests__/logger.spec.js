const createConsumer = require('../index')

const { createCluster, newLogger } = require('testHelpers')

describe('Consumer', () => {
  it('gives access to its logger', () => {
    expect(
      createConsumer({
        cluster: createCluster(),
        groupId: 'test-consumer',
        logger: newLogger(),
      }).logger()
    ).toMatchSnapshot()
  })
})
