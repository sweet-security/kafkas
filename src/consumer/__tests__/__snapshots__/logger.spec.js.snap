// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Consumer gives access to its logger 1`] = `
Object {
  "debug": [Function],
  "error": [Function],
  "info": [Function],
  "namespace": [Function],
  "setLogLevel": [Function],
  "warn": [Function],
}
`;
